db/migrate/20180727211805_create_form526_opt_in.rb

```diff
@@ -0,0 +1,10 @@
+class CreateForm526OptIn < ActiveRecord::Migration
+  def change
+    create_table :form526_opt_ins do |t|
+      t.string :user_uuid, null: false
+      t.string :encrypted_email, null: false
+      t.string :encrypted_email_iv, null: false
```

Comment:
What is this used for?

Reply:
the `_iv` is required for the initialization vector for the cypher text (the encryption).

Reply:
Cool, thanks for clarifying!